src/com/spikes2212/genericsubsystems/drivetrains/commands/OrientWithPID.java

```diff
@@ -0,0 +1,78 @@
+package com.spikes2212.genericsubsystems.drivetrains.commands;
+
+import java.util.function.Supplier;
+
+import com.spikes2212.genericsubsystems.drivetrains.TankDrivetrain;
+import com.spikes2212.utils.PIDSettings;
+
+import edu.wpi.first.wpilibj.PIDController;
+import edu.wpi.first.wpilibj.PIDSource;
+import edu.wpi.first.wpilibj.Timer;
+
+/**
+ * This command is used to turn {@link TankDrivetrain} to a specific degree
+ * using wpilib's {@link PIDController}.
+ *
+ * @see TankDrivetrain
+ * @see PIDController
+ *
+ * @author Simon "C" Kharmatsky
+ */
+public class OrientWithPID extends DriveArcadeWithPID {
+
+	private double lastTimeOnTarget = 0;
+
+	/**
+	 * This constructs new {@link OrientWithPID} using {@link PIDSource},

```

Comment:
note - can't you just explain that it uses the DriveArcadeWithPID command with 0 at movement and just reference the docs in there?